#include <iostream>

using namespace std;

int main(int argc, char *argv[])
{
    throw "This is an exception";
}